### ISSUE TITLE
Why is the Gitlab integration unavailable ?

### ISSUE THREAD

**Issue by zedtux** [Stepsize/atom-better-git-blame#39]
Can you explain why and when it could be available again ? Thank you.

**Comment by nomeyer** [comment]
Hey @zedtux, thanks for enquiring about this.

We don't have plans to add support for a GitLab integration in the near term.

We used to support it because we leveraged the GitLab integration we built for our core product at Stepsize. We've since changed our core product which required changing the way the GitLab integration worked. Unfortunately, we didn't have the resources to make our new integration work with `atom-better-git-blame` and also had to phase out the existing integration.

Although we're still providing support for this package, we're not adding any new functionality at this time.

I'd love to give you pointers on how to implement the GitLab integration but unfortunately this involves changes to our closed source backend, so it'll have to wait.

A more positive note to close – Stepsize is still working on solving the same high level problem for developers and build tools for us to efficiently retrieve information about our code. We're currently focusing on our browser application but plan to build editor integrations as soon as it makes sense (best guess is some time in the first half of next year).

✌️
